packages/api/src/repository.ts

```diff
@@ -80,6 +80,10 @@ export class Repository<T> {
     return await this.collection.find(filter).toArray()
   }
 
+  public async getSortedByName(filter: Filter<T>): Promise<Array<WithId<T>>> {
+    return await this.collection.find(filter, { sort: { name: 1 } }).toArray()
+  }
+
```

Comment:
Could not be necessary now, maybe it would be more useful sorted by bufficorn index

Reply:
I have replace this function with a more generic one